packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -123,17 +137,17 @@ class ButtonGroup extends React.Component {
 
     const allButtons = children ? [] : undefined;
     // eslint-disable-next-line no-nested-ternary
-    const btnRole = onChange ? isMultiSelect ? 'checkbox' : 'radio' : 'button';
+    const btnRole = onChange ? isMultiSelect ? 'checkbox' : 'radio' : '';
```

Comment:
Let's add parentheses here for clarity:
```suggestion
    const btnRole = onChange ? (isMultiSelect ? 'checkbox' : 'radio') : '';
```

Reply:
Added parentheses